README.md

```diff
@@ -74,69 +73,58 @@ We are using macOS to build and run everything locally for now.
    ./requirementsCheck.sh
    ```
 
-   > **NOTE:** Be sure to give permission for the script to execute.
+   > **NOTE:** Be sure to give permission for the script to execute.<br/>
+
+   ```bash
+   sudo chmod +x requirementsCheck.sh
+   ```
 
 3. Install missing dependencies.
 
-4. Switch to **frontend/smart-kickers-game** and run npm install.
+4. In folder [`SmartKickers`](SmartKickers) run docker-compose build.
 
    ```bash
-   cd frontend/smart-kickers-game
-   npm install
+   docker-compose build
```

Comment:
We don't need additional build command as `docker-compose up` automatically build image if it's not present to pull.